app/src/org/commcare/utils/StorageUtils.java

```diff
@@ -110,8 +110,6 @@ public int compare(Integer lhs, Integer rhs) {
                 dateAsSeconds = Long.valueOf(dateAsString);
             } catch (NumberFormatException e) {
                 // Go with the next best ordering for now
-                Logger.log(AndroidLogger.TYPE_ERROR_ASSERTION,
```

Comment:
Hey ,

This definitely stay as a soft assert now, since we've fixed the problem as far as we know.